handler/name.go

```diff
@@ -0,0 +1,16 @@
+package handler
+
+import (
+	"fmt"
+	"strings"
+)
+
+func Name(r Interface) string {
+	split := strings.Split(fmt.Sprintf("%#v", r), ".")
```

Comment:
```suggestion
	split := strings.Split(fmt.Sprintf("%T", r), ".")
```

Reply:
I would keep it as is. I do not mind doing it one way or another but I do not see the point adding more work to what works. 

```
--- FAIL: Test_Handler_Name (0.00s)
    --- FAIL: Test_Handler_Name/0 (0.00s)
        name_test.go:40: expected handler name `foo` got `*foo`
    --- FAIL: Test_Handler_Name/1 (0.00s)
        name_test.go:40: expected handler name `bar` got `*bar`
```

Reply:
Or do you have a good reason for your suggestion?